src/core/firebase.app.module.ts

```diff
@@ -1,40 +0,0 @@
-import { InjectionToken, } from '@angular/core';
-import { FirebaseAppConfig } from './';
-import firebase from '@firebase/app';
-
-import { FirebaseApp as FBApp } from '@firebase/app-types';
-import { FirebaseAuth } from '@firebase/auth-types';
-import { FirebaseDatabase } from '@firebase/database-types';
-import { FirebaseMessaging } from '@firebase/messaging-types';
-import { FirebaseStorage } from '@firebase/storage-types';
-import { FirebaseFirestore } from '@firebase/firestore-types';
-
-export const FirebaseAppConfigToken = new InjectionToken<FirebaseAppConfig>('FirebaseAppConfigToken');
-
```

Comment:
I'm not a fan of removing the`AngularFireModule` in place of making people do their own providers. I also like having the `FirebaseApp` as an injectable.